### ISSUE TITLE
Secure messaging error message content support from VA Mobile App

### ISSUE THREAD

**Issue by mistymg** [department-of-veterans-affairs/va.gov-team#66458]
## About your team

- Team name: VA Mobile App (VA Health and Benefits mobile app)
-	OCTO-DE product owner: Rachel Han
-	Product manager: Adam Kasmier
-	Designer: Multiple
- Accessibility specialist: Brea Blackwelder
- Content writer: Misty Milliron-Grant
-	Slack channel: va-mobile-app
 
### Describe what you need in a few sentences:
There is a known issue in which mobile app users with Premium MHV accounts aren't able to access secure messaging in the app. While a long-term solution is prioritized, the mobile app team is being asked to create an error message to show users when they experience this issue. From stakeholders, this issue will likely also affect VA.gov (with the MHV on VA.gov transition) since it's using the same APIs. We were told by our PO (Rachel Han) that Danielle Thierry is aware of this issue and to engage with her to write a message that can be used across VA.gov and mobile app. We've also received some information that there may be content/messaging about this issue somewhere out there, but we've had no luck in tracking that down from anyone at MHV.

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [X] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [X] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [X] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** to write an error message that will work for both VA.gov and mobile app.
- **Provide date if available**: xx/xx/xxxx

#### Timing for your next step: 
  - [X] Next week (Could be within the next week or 2. I'll also have to work within my own capacity to get this done.)
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [X] No

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [X] No
 

### When do you expect to launch your product to 100% of users?
We'll launch this error message in the app as soon as it final to prevent users from experiencing a dead-end with this tool.
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [X] No

  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- [**Mobile App's epic for this issue**:](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/6852): 
- **[Rachel Han's Slack update](https://dsva.slack.com/archives/C018V2JCWRJ/p1695823142584609)**: Which includes Word doc on problem/solutions
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [X] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [X] I’d like to schedule time to talk about this.
- [ ] Let’s work async and meet as needed.
- [X] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

**Given the confusion/complexity of the issue, it'd be best to have an actual call to discuss it all first.**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next. 

```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/66563
```

**Comment by coforma-terry** [comment]
**CAIA Note ** 

@strelichl will coordinate with @DanielleThierryUSDSVA to craft this messaging.
Added Sitewide Content Label.

**Comment by laurwill** [comment]
Update: We've reached out to Ngozi to ask about existing MHV versions of this, and are meeting with Misty 10/25 to collaborate.

**Comment by laurwill** [comment]
Misty confirmed the mobile app team found a backend fix for this problem — closing this out.
